votes/initiateNewVote/_EDIT_ME.yml

```diff
@@ -3,7 +3,7 @@
 
 # 1. Select a subject for the vote. This can be a question addressed to the TSC
 #    voting members.
-subject: REPLACEME
+subject: Corepack
```

Comment:
The title of the file needs to be changed I guess?

Reply:
Not according to the instructions, which do not mention changing the file name at all. I'm assuming that's taken care of by the github action that actually initiates the vote from this PR. @aduh95 could probably confirm for sure.

Reply:
It needs to be named `_EDIT_ME.yml`, that's the only path that's being "watched" by the workflow: https://github.com/nodejs/TSC/blob/2bfba10edac3ce229db8401a032b604c9c2e7cf0/.github/workflows/initiateNewVote.yml#L4-L7
We could change it to "watch" e.g. any `.yml` file, but not sure it's worth the effort.